fixtures/builder.py

```diff
@@ -40,6 +41,12 @@ def __init__(self, name):
 
         self.repository.status()
 
+    def __del__(self):
+        # Create a hash for the generated fixture.
+        file_contents = dirhash(self.dir, 'sha256')
```

Comment:
Nit: `file_contents` is an unnecessary variable.

Reply:
If we get rid of the var and use
 
 `hash_file.write(dirhash(self.dir, 'sha256'))`
 
 This will actually change the hashes because it hashes the directory *after* we open the empty `hash.txt` file. I tested this to be sure.
 
 Changed the to `hash`. I think it is better to hash the directory with just the fixtures and not the empty `hash.txt` file.

Reply:
Oooh, interesting! Can you add a comment explaining this, so that we don't accidentally change it later?

Reply:
Added a comment